kevm

```diff
@@ -157,33 +188,48 @@ run_get_failing() {
 # ----
 
 run_command="$1" ; shift
+backend="ocaml"
+params=()
+while [[ $# -gt 0 ]]
+do
+    case "$1" in
+        --backend)
+                backend="$2"
+                shift 2
+                ;;
+        *)
+            params+=("$1")
+            shift
+            ;;
+    esac
+done
+
+set -- "${params[@]}"
```

Comment:
This seems more complicated than we need. How about we *only* check if `--backend` is supplied as the *first* argument, and otherwise assume it's not supplied at all? Then can be done with an `if [[ $# -gt 0 ]] && [[ $1 == '--backend' ]]` check.